examples/cfq/input_pipeline.py

```diff
@@ -144,10 +144,12 @@ def get_example_length(self, example: ExampleType) -> tf.Tensor:
       example_len = tf.math.maximum(question_len, query_len)
     return example_len
 
-  def indices_to_sequence_string(self, indices: jnp.ndarray) -> Text:
+  def indices_to_sequence_string(self,
+                                 indices: jnp.ndarray,
+                                 keep_padding: bool = True) -> Text:
```

Comment:
Why not make `keep_padding` False by default? (seems to be what you usually want, and then you don't have to give it as an argument below).

Reply:
will change